server/app/views/applicant/PreventDuplicateSubmissionView.java

```diff
@@ -0,0 +1,91 @@
+package views.applicant;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.h1;
+import static j2html.TagCreator.p;
+
+import controllers.applicant.routes;
+import j2html.tags.specialized.DivTag;
+import java.util.Optional;
+import javax.inject.Inject;
+import play.i18n.Messages;
+import play.mvc.Http.Request;
+import play.twirl.api.Content;
+import services.MessageKey;
+import services.applicant.ApplicantPersonalInfo;
+import services.applicant.ApplicantPersonalInfo.Representation;
+import services.applicant.ReadOnlyApplicantProgramService;
+import views.ApplicationBaseView;
+import views.HtmlBundle;
+import views.components.ButtonStyles;
+import views.style.ApplicantStyles;
+import views.style.StyleUtils;
+
+/**
+ * Renders a page indicating the applicant has not made any changes and asking them if they would
+ * like to continue editing or exit the application.
+ */
+public final class PreventDuplicateSubmissionView extends ApplicationBaseView {
+
+  private final ApplicantLayout layout;
+
+  @Inject
+  PreventDuplicateSubmissionView(ApplicantLayout layout) {
+    this.layout = checkNotNull(layout);
+  }
+
+  public Content render(
+      Request request,
+      ReadOnlyApplicantProgramService roApplicantProgramService,
+      Messages messages,
+      long applicantId) {
+
+    DivTag content =
+        div()
+            .withClasses(ApplicantStyles.PROGRAM_INFORMATION_BOX)
+            .with(
+                p(messages.at(
+                        MessageKey.TITLE_NO_CHANGES_TO_SAVE.getKeyName(),
+                        roApplicantProgramService.getProgramTitle()))
+                    .withClasses("text-black", "font-bold", "text-xl", "pb-8"))
+            .with(p(messages.at(MessageKey.CONTENT_NO_CHANGES.getKeyName())).withClasses("pb-8"))
+            .with(
+                div()
+                    .withClasses(
+                        "flex", "flex-col", "gap-4", StyleUtils.responsiveSmall("flex-row"))
```

Comment:
nit (non-blocking): I think "flex-col" and "flex-row" cancel each other out here, no?  Since "flex-row" is the default, you could remove both of those classes.

Reply:
I'm pretty sure we need both of these since we want the buttons to be in a column when the screen is small and a row otherwise (I copied this from the [Ineligible Screen](https://github.com/civiform/civiform/blob/3fb9bad36d53d0e0216f8e4255779bde5cd1684e/server/app/views/applicant/IneligibleBlockView.java#L98)). The "flex-col" says the buttons should always be in a column, but then the `responsiveSmall("flex-row")` overrides that to say that any screen size bigger than "small" will have them in a row. Let me know if you understand differently!

Reply:
You're right.  Thanks for explaining!